pkgs/by-name/be/bespokesynth/package.nix

```diff
@@ -23,6 +23,7 @@
   libXScrnSaver,
   libGL,
   libxcb,
+  vst-sdk,
```

Comment:
Correct me if I am mistaken, but doesn't adding this to the input make this package `unfree` regardless of the value of `enableVST2`?

Reply:
No, the package builds (without allowing unfree code) when the option is unset.

That's because the `vst-sdk` argument is filled in by `callPackage`, which doesn't do any license checks, and it's only when `vst-sdk` is evaluated that `mkDerivation` errors out; see `pkgs/stdenv/generic/check-meta.nix`